### PR TITLE
Guidelines cleanup

### PR DESCRIPTION
**Pull request by caldav**
* General cleanup
* New metadata step
* Add author metadata
* no command prompt in snippets

### CODE REVIEW

**Comment by caldav** [comment]
@degville @pmatulis please have a look

File: examples/guidelines-snap-tutorials.md [hunk 1]
```diff
@@ -36,30 +23,95 @@ Each tutorial should:
...
+All tutorials should have the same tone, regardless of the topic. This is why you should complete one or two of the existing tutorials before writing your first one.
 
-In short, this isn’t a teacher/student paradigm, but rather friends sharing some time together. Thus, should use the term **we** as much as possible, like **we have just seen**, **we now understand that…**. However **you** can be use for demonstrating things in the user’s context, like: **edit your `file`**, **your `directory` should look like this:**, **your system** or **you will note**. Prefer the **we** as much as possible though.
+In short, this isn’t a teacher/student paradigm, but rather friends sharing some time together. Thus, the term **we** should be used as much as possible, like **we have just seen**, **we now understand that…**. However **you** can be use for demonstrating things in the user’s context, like: **edit your `file`**, **your `directory` should look like this:**, **your system** or **you will note**.
```
Comment (degville) [inline examples/guidelines-snap-tutorials.md]: However, **you** can be used, not *use*.

File: examples/guidelines-snap-tutorials.md [hunk 2]
```diff
@@ -36,30 +23,95 @@ Each tutorial should:
...
+Duration: 2:00
+
+Each tutorial source file starts with a set of metadata that will be consumed by tutorials.ubuntu.com for presentation purposes.
+
+For example, at the top of the source file of this tutorial:
+
+```
+---
+id: tutorial-guidelines
+summary: A summary tutorial description. Remember the content of it (as the title) is searchable. It should be around 26 words.
```
Comment (degville) [inline examples/guidelines-snap-tutorials.md]: (*and* the title)?

File: examples/guidelines-snap-tutorials.md [hunk 3]
```diff
@@ -36,30 +23,95 @@ Each tutorial should:
...
+ * Remember that **tags** are searchable
+
+
+ * **difficulty** spans from 1 to 5. Beginners without previous knowledge of the given topic should be able to follow and understand tutorials from level 1 without any other prerequisite knowledge. As a guide:
+
+     1. Complete beginner with Ubuntu - just about knows how to open a terminal
+     * Ubuntu novice - can be trusted to enter commands but experience limited to simple file operations.
+     * Experienced user - doesn't need explanations about common topics (networking, sudo, services)
+     * Advanced user - has experience running Ubuntu for many years, but may be unfamiliar with some sysadmin/programming topics.
+     * Ubuntu sysadmin/developer- very familiar with system and can be expected to know details about
```
Comment (degville) [inline examples/guidelines-snap-tutorials.md]: inconsistant use of full stops (.) in this list.

Comment (degville) [inline examples/guidelines-snap-tutorials.md]: very familar with *the* system*? Also, this is a ltitle vague. Perhaps very familiar with all aspects of the operating system ...

File: examples/guidelines-snap-tutorials.md [hunk 4]
```diff
@@ -36,30 +23,95 @@ Each tutorial should:
...
+
+### Title
+
+The title should be set as the first heading of your file.
+
+```markdown
+# Ubuntu tutorial guidelines
+```
+
+Tutorial title should be kept short (5 words as a guide) so as not to break design. Do make the titles concise, but specific where possible - e.g. "Create a bootable USB stick on Windows 10"
```
Comment (degville) [inline examples/guidelines-snap-tutorials.md]: I think I'd rewrite this; Tutorial title should be kept short (5 words as a guide) to not break the design. Try to make the titles concise but also specific, e.g. "Create a bootable USB stick on Windows 10"

File: examples/guidelines-snap-tutorials.md [hunk 5]
```diff
@@ -36,30 +23,95 @@ Each tutorial should:
...
+
+### Steps
+
+Each step is delimited by a second level title, for example:
+
+```markdown
+## Step title
+```
+
+It should have a step duration, following immediately the step. The total tutorial time will then be computed automatically. A third level heading or empty line will break into the step content.
```
Comment (degville) [inline examples/guidelines-snap-tutorials.md]: How about `A step duration should immediately follow the step`. Also, what units for time?

File: examples/guidelines-snap-tutorials.md [hunk 6]
```diff
@@ -36,30 +23,95 @@ Each tutorial should:
...
+This is the **Overview** step. It should be the first step of every tutorial. This first paragraph is a summary of the tutorial's objectives, stating what the reader will learn by completing it. An image can be included, as well as [external links](https://tutorial.ubuntu.com).
 ![Ubuntu logo](./ubuntu-logo.svg)
 
 ### What you'll learn
 
-- This section layouts in a few bullet points what each step will teach to the attendee.
-- It is important to state clearly what we think people can expect from a tutorial to ensure it matches their intents.
-- As you can see, this is presented as task list bullets.
+- This section includes a list of steps that will be needed to achieve the objective.
+- State things clearly so that the reader's intents and the tutorial objectives correspond.
```
Comment (degville) [inline examples/guidelines-snap-tutorials.md]: *intent*, but I think this is a little unclear. Maybe *reader's expectations*?

File: examples/guidelines-snap-tutorials.md [hunk 7]
```diff
@@ -91,16 +156,16 @@ From the tutorial directory, you can iterate manually on a markdown file using:
 
 ### Advice and general rules
 
-In addition to the previous points on what a tutorial should be and what is mandatory, you should take special attention to some points:
+In addition to the previous points on what a tutorial should be and what is mandatory, you should  pay special attention to some points:
 
 #### Each step should be concise, but not too short
 
-Pay special attention to the step size. They shouldn’t take more than 7 minutes each. Don’t make them too short either (like a few lines to read). Of course, some steps will be shorter than others like first and last ones.
+Be wary of a step's length. Each one shouldn’t take longer than 7 minutes to complete. Don’t make them too short either. Naturally, some steps will be shorter than others (like first and last one).
```
Comment (degville) [inline examples/guidelines-snap-tutorials.md]: (such as the first and last steps)?

File: examples/guidelines-snap-tutorials.md [hunk 8]
```diff
@@ -179,7 +254,7 @@ Syntax is regular [markdown syntax](https://guides.github.com/features/mastering
 
 ### Text
 
-For coherence in text inside code snippets, notes and survey with paragraphs, each end of line is reflected. A single end of line create a line return (*br*), while an end of line followed by an empty line will delimit pagraphs.
+For coherence in text inside code snippets, notes and survey with paragraphs, each end of line is reflected. A single end of line create a line return (*br*), while an end of line followed by an empty line will delimit paragraphs.
```
Comment (degville) [inline examples/guidelines-snap-tutorials.md]: To ensure the coherence of text inside code snippets, notes and a survey with paragraphs, how you end a line within these sections will affect the output. A single end of line will create a line return (*br*), while an end of line followed by an empty line will delimit paragraphs.

**Comment by evilnick** [comment]
Yes, thanks everyone!

**Pull request merged**